### ISSUE TITLE
What is it/how does it work?

### ISSUE THREAD

**Issue by balk77** [yahat/home_assistant_mqtt_translation#1]
Hi, I am looking for a way to publish Z-Wave events created in Home Assistant to MQTT. Is this script capable of doing this? And how do I use it? Thanks for sharing!

**Comment by vladbabii** [comment]
Yes, it creates mqtt topics with information from home assistant. I'll update the readme in the next 24-48 hours with instructions & examples.

**Comment by larstobi** [comment]
@vladbabii Hoping for the examples. :-D 

Are you still using this? How is it working for you?

**Comment by vladbabii** [comment]
@larstobi 
I've used it until a couple of months ago, now i'm using and working on next version which is more user-friendly and can handle multiple home assistant instances and some filtering (only publish what you need). If this is too complicated for you, I'd suggest installing node red and the home assistant plugin or publish to mqtt from home assistant and use node red mqtt component to do more stuff. What is your end-goal ?
